engine/overworld/map_objects.asm

```diff
@@ -2884,14 +2900,17 @@ InitSprites:
 	ld a, [hli]
 	ld h, [hl]
 	ld l, a
-	ldh a, [hUsedSpriteIndex]
+	ldh a, [hUsedOAMIndex]
+	ld c, a ; no-optimize a = N - a (c gets used below)
```

Comment:
This `c` doesn't get used below, because the value of `SPRITEOAMSTRUCT_LENGTH * (NUM_SPRITE_OAM_STRUCTS - 1) - [hUsedOAMIndex]` immediately gets assigned to `c`.